webapp/src/components/PollDetailPage/PollDetailPage.tsx

```diff
@@ -299,6 +299,8 @@ export default class PollDetailPage extends React.PureComponent<Props, State> {
                       totalPages={totalPages}
                       firstItem={null}
                       lastItem={null}
+                      nextItem={null}
+                      prevItem={null}
```

Comment:
I only just catch this. Is it necessary? I would have5 imagined that not adding those props yields the same result

Reply:
Not adding these lines result in the arrows (next/prev) showing up, but they break on mobile causing horizontal scroll. We can't reduce the number of pages due to limitations from the underlying `<Pagination>` component (sematic-ui), so we decided to get rid of the arrows for now (we should address this at `decentraland-ui` level to fix it the right way).

This was already done in PollsTable, but was missing here, so I added it to keep both paginations consistent.